fragalysis/settings.py

```diff
@@ -37,7 +37,7 @@
 )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
Make sure you don't ever commit DEBUG=True when merging.

Reply:
yes of course...but without gitignore it is somesimes difficult :)